### ISSUE TITLE
[Bug] wxGUI: d.histogram not shown with Python 3.10

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass#2047]
**Describe the bug**

Loading a raster map and selecting "Analyse map" > "Create histogram with d.histogram" does not show it:

![image](https://user-images.githubusercontent.com/1295172/147598414-d8b1dba5-d446-4ca1-8934-fe63e8070635.png)

Error:

```
Traceback (most recent call last):
  File "/usr/lib64/python3.10/site-packages/wx/core.py",
line 3285, in <lambda>

lambda event: event.callable(*event.args, **event.kw) )
  File "/home/mneteler/software/grass80/dist.x86_64-pc-
linux-gnu/gui/wxpython/modules/histogram.py", line 395, in
OnOptions

module.ParseCommand(cmd, completed=(self.GetOptData, None,
self.params))
  File "/home/mneteler/software/grass80/dist.x86_64-pc-
linux-gnu/gui/wxpython/gui_core/forms.py", line 3136, in
ParseCommand

self.mf = TaskFrame(
  File "/home/mneteler/software/grass80/dist.x86_64-pc-
linux-gnu/gui/wxpython/gui_core/forms.py", line 567, in
__init__

self.notebookpanel = CmdPanel(
  File "/home/mneteler/software/grass80/dist.x86_64-pc-
linux-gnu/gui/wxpython/gui_core/forms.py", line 1424, in
__init__

win = SpinCtrl(
  File "/home/mneteler/software/grass80/dist.x86_64-pc-
linux-gnu/gui/wxpython/gui_core/wrap.py", line 161, in
__init__

wx.SpinCtrl.__init__(self, *args, **kwargs)
TypeError
:
SpinCtrl(): arguments did not match any overloaded call:
  overload 1: 'parent' is not a valid keyword argument
  overload 2: argument 'min' has unexpected type 'float'
```

**To Reproduce**

1. Load elevation map
2. Create histogram
3. See plot with missing histogram

**System description (please complete the following information):**

- Operating System: Fedora 35
- affects both G8.0 + G.7.8

```
GRASS version: 7.8.7dev                                                         
Code revision: f9f30f33f                                                        
Build date: 2021-12-14                                                          
Build platform: x86_64-pc-linux-gnu                                             
GDAL: 3.3.2                                                                     
PROJ: 8.2.0                                                                     
GEOS: 3.9.2                                                                     
SQLite: 3.36.0                                                                  
Python: 3.10.1                                                                  
wxPython: 4.0.7                                                                 
Platform: Linux-5.15.6-200.fc35.x86_64-x86_64-with-glibc2.34
```

Note: this is wxPython: 4.0.7-23 (i.e., with multiple recent patches to make the wxGUI start).

**Comment by neteler** [comment]
Thanks to PR #2050 this works again in G8.0.

A backport of #2050 will likely fix it in G7.8.7 as well, keeping issue open.

**Comment by neteler** [comment]
Problem solved in ecd13587d7ccde56b9f443ee884870076983c471
